### PR TITLE
Image input

### LINKED ISSUE #3: Take a photo
As a user,
I like to take a photo of a printed document,
so that the OCR can extract the text of it.

### LINKED ISSUE #4: Save a document
I like to save a document,
so I can access them later.

A document contains:

the image
the text read from OCR
a name

### PR DESCRIPTION
**Pull request by AnkePfei2fer**
image

fixes #3 
fixes #4

### CODE REVIEW

File: src/app/components/imageInput/imageInput.tsx [hunk 1]
```diff
@@ -11,24 +15,28 @@ function ImageInput() {
...
+            className={styles.input}
+            type="file"
+            accept="image/*"
+            onChange={handleChange}
+          ></input>
+          <h1 className={styles.inputText}>Import a document</h1>
+          <img
+            className={styles.inputImage}
+            src="../src/assets/empty-folder-farbig.png"
+          ></img>
```
Comment (lajanzen) [inline src/app/components/imageInput/imageInput.tsx]:
```suggestion
          />
```

**Review by lajanzen — APPROVED** [review]
🚀

File: src/app/components/AddDocumentForm/AddDocumentForm.module.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+.saveButton {
+  background: var(--color-secondary);
+  border-radius: 10px;
+  color: var(--color-text-primary);
+  font-size: 1.25rem;
+  text-align: center;
+  font-weight: 800;
+  padding: 0.5rem;
+  width: 200px;
```
Comment (lmachens) [inline src/app/components/AddDocumentForm/AddDocumentForm.module.css]: A fixed width for a button is not a good idea. If you increase the root font-size or change the text, you will face an overflow.

File: src/app/components/AddDocumentForm/AddDocumentForm.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  return (
+    <form className={styles.saveContainer} onSubmit={handleSubmit}>
+      <input
+        type="text"
+        className={styles.filenameInput}
+        placeholder="Enter file name"
+        value={title}
+        onChange={(event) => setTitle(event.target.value)}
+      ></input>
```
Comment (lmachens) [inline src/app/components/AddDocumentForm/AddDocumentForm.tsx]:
```suggestion
      />
```

**Pull request merged**